scripts/dts/edtlib.py

```diff
@@ -553,6 +553,9 @@ def _init_prop(self, name, options):
         # Handle enum checking
         enum = options.get("enum")
         if enum:
+            if type(enum) is not list:
```

Comment:
`if not isinstance(enum, list)` is more common.

Reply:
done